op-node/rollup/derive/pipeline.go

```diff
@@ -130,6 +132,25 @@ func (dp *DerivationPipeline) UnsafeL2Head() eth.L2BlockRef {
 	return dp.eng.UnsafeL2Head()
 }
 
+func (dp *DerivationPipeline) StartPayload(ctx context.Context, parent eth.L2BlockRef, attrs *eth.PayloadAttributes, updateSafe bool) (errType BlockInsertionErrType, err error) {
+	return dp.eng.StartPayload(ctx, parent, attrs, updateSafe)
+}
+
+func (dp *DerivationPipeline) ConfirmPayload(ctx context.Context) (out *eth.ExecutionPayload, errTyp BlockInsertionErrType, err error) {
+	return dp.eng.ConfirmPayload(ctx)
+}
+
+func (dp *DerivationPipeline) CancelPayload(ctx context.Context, force bool) error {
+	return dp.eng.CancelPayload(ctx, force)
+}
+
+func (dp *DerivationPipeline) BuildingPayload() (onto eth.L2BlockRef, id eth.PayloadID, safe bool) {
+	return dp.eng.BuildingPayload()
+}
+
+// SetUnsafeHead changes the forkchoice state unsafe head, without changing the engine.
+//
+// deprecated: use the EngineControl interface instead.
```

Comment:
What do you think about logging a deprecation warning here? To make it obvious if this is being called still

Reply:
It's getting removed in the PR 3 in the stack, and don't think we really need to do a lot around deprecation here since it's not used something someone external would depend on. Mostly just putting it here in case that later PR in the stack takes a while to land.